### PR TITLE
Bug/41

### PR DESCRIPTION
**Pull request by mbovo**
Solve the issue #41 
@fntlnz as you reported, i've updated using the workaround suggested moby/moby/issues/33989
This break vendoring at this time. Running again dep ensure will replace the docker/docker lib with version 17.5.0-ce-rc3

### CODE REVIEW

File: provider/swarm.go [hunk 1]
```diff
@@ -39,7 +39,10 @@ func (p SwarmProvider) Name() string {
 
 func (p SwarmProvider) Scale(serviceId string, target int, direction bool) error {
 	ctx := context.Background()
-	service, _, err := p.dockerClient.ServiceInspectWithRaw(ctx, serviceId)
+	// Correct bug #41 using docker/docker v17.06.1-ce-rc4
+	siopts := types.ServiceInspectOptions{}
+	siopts.InsertDefaults = true
```
Comment (fntlnz) [inline provider/swarm.go]: I would avoid instantiating and then assigning `siopts` here.
It's just not needed anywhere else, it even fits 120 columns!

Comment (fntlnz) [inline provider/swarm.go]: Please add a comment like:

```
// Service inspect returns a service showing default values in empty fields
```

I had to actually execute the code to understand how the struct was filled, I don't want to do that too in the future :D

**Comment by fntlnz** [comment]
@mbovo worth using v17.06.1-ce ?

@gianarb reviewing this PR evidences that we are missing a lot of testing !

**Comment by fntlnz** [comment]
@mbovo we needed to address this so @gianarb created a PR on top of your work keeping your commits. https://github.com/gianarb/orbiter/pull/46

**Comment by gianarb** [comment]
thanks @mbovo !https://github.com/gianarb/orbiter/pull/46

**Comment by mbovo** [comment]
No problem, was afk for a while in the last days

**Comment by gianarb** [comment]
Sorry, I was happy to have it merged asap! :heart:

**Pull request closed without merging**